src/App.tsx

```diff
@@ -17,18 +37,33 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                todos={todos}
+                setFilteredTodos={setFilteredTodos}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading || todos.length === 0
```

Comment:
```suggestion
              {isLoading || !todos.length
```

can be a bit simplify